docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/test-api/assertions/
+checked: false
+---
+# Assertions
+
+To check if the state of the tested webpage matches the one you expect to see, use *assertions*.
+
+We will provide a comprehensive [set of assertions](assertion-api.md) and all of them based on Behavior Driven Development style (BDD-style).
+The purpose of BDD style is writing tests that reflect the behavior and identify appropriate tests to writing.
+
+This topic contains the following sections.
+
+* [Assertion timeout](#assertion-timeout)
+* [Assertion options](#assertion-options)
+  * [options.timeout](#optionstimeout)
+
+The following example shows how to check the two values for equality:
+
+```text
+    await t.expect(actual).eql(expected, message, options);
+```
+
+Parameter              | Type     | Description
+---------------------- | -------- | -------------------------------------------------------------------------------
+`actual`                 | Any type | The comparison value
+`expected`                 | Any type | Expected value
+`message`&#160;*(optional)* | String   | Assertion message, will be indicated in the report in case of error.
+`options`&#160;*(optional)* | Object   | See [Options](#assertion-options).
+
+Assertions allow you check values of the standard types (String, Number, Object and etc.) and also assertions can be used
+to check state of the webpage's elements.
+In order to check state of any element on the page in assertion you can use properties and methods of
```

Comment:
To check state of any element on the page in assertion, you can use properties and methods of